packages/xrpl/test/models/permissionedDomainSet.test.ts

```diff
@@ -0,0 +1,104 @@
+import { stringToHex } from '@xrplf/isomorphic/dist/utils'
+import { assert } from 'chai'
+
+import { AuthorizeCredential, validate, ValidationError } from '../../src'
+import { validatePermissionedDomainSet } from '../../src/models/transactions/permissionedDomainSet'
+
+/**
+ * PermissionedDomainSet Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('PermissionedDomainSet', function () {
+  let tx
+  const sampleCredential: AuthorizeCredential = {
+    Credential: {
+      CredentialType: stringToHex('Passport'),
+      Issuer: 'rfmDuhDyLGgx94qiwf3YF8BUV5j6KSvE8',
+    },
+  }
+
+  beforeEach(function () {
+    tx = {
+      TransactionType: 'PermissionedDomainSet',
+      Account: 'rfmDuhDyLGgx94qiwf3YF8BUV5j6KSvE8',
+      DomainID:
+        'D88930B33C2B6831660BFD006D91FF100011AD4E67CBB78B460AF0A215103737',
+      AcceptedCredentials: [sampleCredential],
+    } as any
+  })
+
+  it('verifies valid PermissionedDomainSet', function () {
+    assert.doesNotThrow(() => validatePermissionedDomainSet(tx))
+    assert.doesNotThrow(() => validate(tx))
+  })
+
+  it(`throws with invalid field DomainID`, function () {
+    // DomainID is expected to be a string
+    tx.DomainID = 1234
+    const errorMessage = 'PermissionedDomainSet: invalid field DomainID'
+    assert.throws(
+      () => validatePermissionedDomainSet(tx),
+      ValidationError,
+      errorMessage,
+    )
+    assert.throws(() => validate(tx), ValidationError, errorMessage)
+  })
+
+  it(`throws w/ missing field AcceptedCredentials`, function () {
+    delete tx.AcceptedCredentials
+    const errorMessage =
+      'PermissionedDomainSet: missing field AcceptedCredentials'
+    assert.throws(
+      () => validatePermissionedDomainSet(tx),
+      ValidationError,
+      errorMessage,
+    )
+    assert.throws(() => validate(tx), ValidationError, errorMessage)
+  })
+
+  it('throws when AcceptedCredentials exceeds maximum length', function () {
```

Comment:
All the tests should check `validatePermissionedDomainSet` and `validate`

Reply:
`validate` encompasses `validatePermissionedDomainSet`. Thanks, I've decided to only invoke the former method.

Should I make this change inside the PDSet model verification too? `validate` method multiplexes the validations for all transactions

Reply:
[2080c21](https://github.com/XRPLF/xrpl.js/pull/2874/commits/2080c21bd425a596f844d2898ed65c465f19adfd)

Reply:
See other test files - it's better to check both, just in case.

Reply:
I don't see the need to separately test `validatePermissionedDomainSet` over and above the `validate` method. What do you think?

Yes, other test files invoke both the methods, but that appears unnecessary.

Reply:
It's easy to check both at the same time, and doesn't really cost anything, so we should be testing both. This has the potential to catch issues if there _are_ discrepancies.